test/test-applications/integrations/Samples.AWS.SQS/Samples.AWS.SQS.csproj

```
@@ -7,7 +7,7 @@
   </PropertyGroup>
```

I think this project requires `<LoadManagedProfilerFromProfilerDirectory>true</LoadManagedProfilerFromProfilerDirectory>` so we can run this locally:
```suggestion
    <LoadManagedProfilerFromProfilerDirectory>true</LoadManagedProfilerFromProfilerDirectory>
  </PropertyGroup>
```

---

Done in https://github.com/DataDog/dd-trace-dotnet/pull/1726/commits/c2ddd2e6b969ca2a7d4e8506f2cbf947460a9c3e